src/views.js

```diff
@@ -0,0 +1,50 @@
+export default class View {
+  static resultDiv() {
+    const results = document.getElementById('results');
+    const h1 = document.createElement('h1');
+    h1.id = 'tempH1';
+    const h2 = document.createElement('h2');
+    results.innerHTML = '';
+    return [results, h1, h2];
+  }
+
+  static changeBackground(response) {
+    const [results,, h2] = View.resultDiv();
+    const temp = parseInt(response.main.temp, 10) - 273;
+
+    if (temp > 30) {
+      document.body.style.backgroundColor = '#FD6468';
+      h2.textContent = '🧊🍹👕🩳';
+    } else if (temp > 10) {
+      document.body.style.backgroundColor = 'lightskyblue';
+      h2.textContent = '😎🍹👕🩳';
+    } else if (temp < 10) {
+      document.body.style.backgroundColor = 'lightskyblue';
```

Comment:
- [ ] [OPTIONAL] Keep all the styling logic in CSS files (If you need to change something like a background image, add the classes with the different backgrounds to the CSS and then change the class name attribute with JavaScript) (screenshot from Pathwright)

Reply:
Added to future improvements. ✔️